### DISCUSSION TITLE
Naked gopro 10 no backscreen wifi connect

### DISCUSSION THREAD

**Discussion by fernandommartins84** [gopro/labs#770] [Q&A]
I have built a naked gopro from a hero 10 I bought without back screen. Is there anyway I can connect on mobile app using gopro labs? Just to be clear that I don't have the backscreen. It is working ok but I would like to connect to my phone, if possible.

**Comment by dnewman-gpsw** [comment]
If you have Labs pre installed just use the !PA command.

**Reply by fernandommartins84** [reply]
I tried but looks like its not recognizing the command. Gopro reads the qr code, I press mode 3 times but nothing happens. Just changes the mode 3 times.

**Reply by dnewman-gpsw** [reply]
Presses are only required for camera with screens.  Naked camera with Labs, or GoPro Bones, no button presses are need.  Scan then do the rest with Quik.

**Reply by fernandommartins84** [reply]
My first attempt was without pressing anything. And also not finding the
cam on the app. When cam goes to sync mode, is there anything shown on
small front screen? Nothing is changing there.

**Reply by mycosd** [reply]
On my naked Hero12 I just experienced something similar, maybe related to this. I couldn't connect it to the App, so I tried pairing and this didn't work either. After a lot of attempts, I could get the pairing started, but once I disconnect the battery it wouldn't re-connect.

Long story short: for some reason when I connect a battery and power it on, bluetooth/wifi doesn't work at all. I have to power down (which takes extremely long) and then power up and only then I can pair/re-connect to the App. After that, powering down will go a lot quicker too. Not sure if this is related to Labs though, it's my first Hero12 which I pushed the resent Labs firmware onto it before stripping it down. I have another Hero12 which isn't stripped down yet and doesn't show this issue.

**Comment by fernandommartins84** [comment]
Thanks mate. Will keep trying here

**Comment by thiagozmed** [comment]
Did the command !PA worked for you @fernandommartins84 ? I'm having the same problem( e sou brasileiro também, se caso tu tenha conseguido resolver e puder me chamar no insta @thiagozlmed agradeço!)
I have a Naked GoPro 10, I bought used, and I'm trying to make the GoPro Quik connect for a better use

**Comment by fernandommartins84** [comment]
Cara, nunca consegui conectar no celular pelo app. So consigo mudar pelo
goprolabs msm. Tentei um milhao de coisas q falaram na internet, mas nao
conecta 😕

**Reply by thiagozmed** [reply]
Puts que coisa maninho. To quase vendo uma tela pra conseguir fazer esse pareamento. A fotometria pelo app é mt melhor, né? Esse GoPro Labs tu bota a config mas fica difícil de saber se a imagem ta na configuração certa praquele cenário. Tem que tirar o cartão dela, jogar pro pc, ver o vídeo e voltar pras configs

**Comment by fernandommartins84** [comment]
Eh demais. Nao curto tb pq nao tenho muita paciencia pra ficar estudando
todas essas configs. Pensei em comprar uma tela tb mas fiquei com medo de
nao funcionar pq vi gente falando q muitas vezes uma tela q nao eh a da
propria gopro nao funciona. Mas pra ser sincero nao fui muito a fundo
nisso. Mas queria pq principalmente poder prever a imagem, seria muito util
pra mim.

**Reply by thiagozmed** [reply]
é, vamos ter que desembolsar e comprar uma naked nova 0km mesmo! Por aqui, tbm nem sinal de conseguir parear ela!

**Reply by thiagozmed** [reply]
Fernando, não sei o q eu fiz aqui mas eu consegui conectar a minha! Se quiser me chamar no insta @thiagozlmed posso tentar te ajudar!

**Comment by fernandommartins84** [comment]
Putz q maravilha. Vou te adicionar la cara. Nossa isso salvaria demais.

Em qua., 19 de fev. de 2025 às 18:35, thiagozmed ***@***.***>
escreveu:
